src/Sfx-Standalone/modules/proxy.object/data-info-manager.ts

```diff
@@ -172,14 +172,18 @@ export class DataInfoManager implements IDisposable {
 
         while (currentObj && currentObj !== Object.prototype) {
             const propertyDescriptors = Object.getOwnPropertyDescriptors(currentObj);
+            const isClass = Function.isFunction(propertyDescriptors["constructor"]);
 
             for (const propertyName in propertyDescriptors) {
                 const propertyDescriptor = propertyDescriptors[propertyName];
 
-                if (!propertyDescriptor.enumerable
-                    || !propertyDescriptor.writable
+                if ((!propertyDescriptor.writable
+                    && !propertyDescriptor.configurable
                     && !propertyDescriptor.get
-                    && !propertyDescriptor.set) {
+                    && !propertyDescriptor.set)
+                    || (isClass 
+                        && !propertyDescriptor.enumerable
+                        && Function.isFunction(propertyDescriptor.value))) {

```

Comment:
what is the purpose of following check ? 
Function.isFunction(propertyDescriptor.value)

Reply:
For members of a class, only cache those which are not enumerable functions, whose values are more stable.